agent/main/test/Frames.test.ts

```diff
@@ -68,27 +60,6 @@ describe('Frames', () => {
       );
     });
 
-    it('should dispose context on navigation', async () => {
```

Comment:
Do we want to actually test what the websockets should be doing in these 2 cases?

Reply:
No really needed since these are all linked to frames and not stored separately. Also websocket are auto cleaned if not responding to ping